src/io/iomanager.cpp

```diff
@@ -10,30 +10,27 @@
 
 #include "lib/logging.hpp"
 
-#include "proto_src/MessageTypes.pb.h"
-#include "proto_src/Console.pb.h"
-#include "proto_src/DirectMode.pb.h"
-#include "proto_src/RealTimeControl.pb.h"
-#include "proto_src/UniverseControl.pb.h"
-#include "google/protobuf/io/zero_copy_stream.h"
-
 #include "lib/macros.hpp"
 #include "net/sock_address_factory.hpp"
 #include <netdb.h>
 
 
+COMPILER_SUPRESS("-Wuseless-cast")
 #include "proto_src/MessageTypes.pb.h"
 #include "proto_src/Console.pb.h"
 #include "proto_src/DirectMode.pb.h"
 #include "proto_src/FilterMode.pb.h"
 #include "proto_src/RealTimeControl.pb.h"
 #include "proto_src/UniverseControl.pb.h"
+COMPILER_RESTORE("-Wuseless-cast")
 #include "google/protobuf/io/zero_copy_stream.h"
 
 #include "io/universe_sender.hpp"
 #include "dmx/ftdi_universe.hpp"
 #include "xml/show_files.hpp"
 
+#include <iostream>
```

Comment:
do we need an iostream? is that usefull for the connection, or was it only for debugging (with cout)?

Reply:
I'll remove it quickly (debugging artefact)